clusters/build-clusters/01_cluster/prow/03_deployment/admin_crier_rbac.yaml

```diff
@@ -29,6 +29,12 @@ items:
       verbs:
         - "get"
         - "list"
+    - apiGroups:
+        - ""
+      resources:
+        - namespaces
+      verbs:
+        - get
```

Comment:
What? Why does it need that?

Reply:
That is used in the process of generating the kubeconfig file.
When oc-login, it has the following error:

```
bash ./clusters/build-clusters/01_cluster/hack/generate-bw-items.sh
Logged into "https://api.build01.ci.devcluster.openshift.com:6443" as "system:serviceaccount:ci:crier" using the token provided.

You don't have any projects. Contact your system administrator to request a project.
...
```

Reply:
I do not have other workaround for this.
Tradeoff between automation and addition RBAC of `get-ns` for crier.

Reply:
Rather than the whole `oc sa get-token` you can simply do a
```
oc serviceaccounts create-kubeconfig crier >$config
```

Big demerit for @stevekuznetsov for not catching that in the initial review

Reply:
Done.